pydoctor/test/test_templatewriter.py

```diff
@@ -69,8 +69,10 @@ def test_basic_package(tmp_path: Path) -> None:
     w.writeDocsFor(root, False)
     w.writeModuleIndex(system)
     for ob in system.allobjects.values():
-        if ob.documentation_location is model.DocLocation.OWN_PAGE:
-            assert (tmp_path / f'{ob.fullName()}.html').is_file()
+        url = ob.url
+        if '#' in url:
```

Comment:
I don't understand this change

Reply:
The old code would construct a file path from the full name. The new code takes the opposite approach and deconstructs the relative URL into a file path, by stripping off the fragment part.